### PR TITLE
Convert Ore to Iron and formate.

### PR DESCRIPTION
**Pull request by azhurauski-sc**

### CODE REVIEW

File: src/utils/oreToIron.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const oreToIron = (ore: bigint): number => {
+  return ore ? Number(ore) / 10000000 : 0
```
Comment (lwisne) [inline src/utils/oreToIron.ts:2]: This is the wrong divisor. One more zero is needed - 100000000

Generally, when doing wallet functionality you should be running the Iron Fish CLI and comparing against that. That is a good source of truth, and we are essentially just building a GUI for that functionality.

Comment (lwisne) [reply src/utils/oreToIron.ts:2]: Even better than knowing the right numbers here would be using the CurrencyUtils from the `ironfish` package. You can access this, I think?

**Review by ndemidovich-sc — CHANGES_REQUESTED** [review]
See comment by @lwisne

File: src/components/AccountBalance.tsx [hunk 2]
```diff
@@ -10,7 +11,8 @@ const AccountBalance: FC<{
 }> = ({
   accountId,
   skeletonProps,
-  renderBalance = balance => balance?.confirmed?.toString() || 0,
+  renderBalance = balance =>
+    CurrencyUtils.renderIron(balance?.confirmed || '0'),
```
Comment (ndemidovich-sc) [inline src/components/AccountBalance.tsx:15]: May be better to use `encodeIron` function (will return 0.1)? `renderIron` will return 8 digits after dot (ex 0.10000000).

**Review by ndemidovich-sc — APPROVED** [review]

File: src/components/AccountBalance.tsx (hunk 2 above)
Comment (azhurauski-sc) [reply src/components/AccountBalance.tsx:15]: As I got from conversation it need 8 decimal places, @lwisne is it correct?

**Pull request merged**